nh_observations/frequencies.py

```diff
@@ -1,20 +1,71 @@
 # -*- coding: utf-8 -*-
 """
 A single place for different frequency values to be read from.
-Values are tuples with the time in minutes and a description.
+Times are in minutes.
 """
-EVERY_15_MINUTES = (15, 'Every 15 Minutes')
-EVERY_30_MINUTES = (30, 'Every 30 Minutes')
-EVERY_HOUR = (60, 'Every Hour')
-EVERY_2_HOURS = (120, 'Every 2 Hours')
-EVERY_4_HOURS = (240, 'Every 4 Hours')
-EVERY_6_HOURS = (360, 'Every 6 Hours')
-EVERY_8_HOURS = (480, 'Every 8 Hours')
-EVERY_10_HOURS = (600, 'Every 10 Hours')
-EVERY_12_HOURS = (720, 'Every 12 Hours')
-EVERY_DAY = (1440, 'Every Day')
-EVERY_3_DAYS = (4320, 'Every 3 Days')
-EVERY_WEEK = (10080, 'Every Week')
+FIFTEEN_MINUTES = 15
+THIRTY_MINUTES = 30
+ONE_HOUR = 60
+TWO_HOURS = 120
+FOUR_HOURS = 240
+SIX_HOURS = 360
+EIGHT_HOURS = 480
+TEN_HOURS = 600
+TWELVE_HOURS = 720
+ONE_DAY = 1440
+THREE_DAYS = 4320
+ONE_WEEK = 10080
+
+# Tuples with the time and a label.
+EVERY_15_MINUTES = (FIFTEEN_MINUTES, 'Every 15 Minutes')
+EVERY_30_MINUTES = (THIRTY_MINUTES, 'Every 30 Minutes')
+EVERY_HOUR = (ONE_HOUR, 'Every Hour')
+EVERY_2_HOURS = (TWO_HOURS, 'Every 2 Hours')
+EVERY_4_HOURS = (FOUR_HOURS, 'Every 4 Hours')
+EVERY_6_HOURS = (SIX_HOURS, 'Every 6 Hours')
+EVERY_8_HOURS = (EIGHT_HOURS, 'Every 8 Hours')
+EVERY_10_HOURS = (TEN_HOURS, 'Every 10 Hours')
+EVERY_12_HOURS = (TWELVE_HOURS, 'Every 12 Hours')
+EVERY_DAY = (ONE_DAY, 'Every Day')
+EVERY_3_DAYS = (THREE_DAYS, 'Every 3 Days')
+EVERY_WEEK = (ONE_WEEK, 'Every Week')
+
+# To summarise the mappings below, frequencies after refusals are capped at
+# 24 hours.
+#
+# Could have just made a simple function but decided to write them out long
+# hand as they may change in the future. Return the tuple with the label rather
+# than just the time so it can be useful in more scenarios.
+PATIENT_REFUSAL_ADJUSTMENTS = {
```

Comment:
Ah actually looking at it again (I originally looked at this due to the issues Nayi was having yesterday) you're just using the first item in the tuple for the key in the dictionary. I think I was under the impression at the time that this was reassigning the value. 

In that case no cause for concern!